### PR TITLE
Client connect

### PR DESCRIPTION
**Pull request by bibas07**
Connect Client - Upload Documents

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { defineConfig } from '@playwright/test';
 export default defineConfig({
     testDir: './src/tests',
     /* Maximum time one test can run for. */
-    timeout: 1 * 60 * 1000,
```
Comment (sirjandrn25) [inline playwright.config.ts:6]: should be add  15 second time

File: src/tests/Vendor_Onboarding/initial_gstvendormanaged_vowithclientconnect.spec.ts [hunk 2]
```diff
@@ -130,54 +91,27 @@ describe('TCCC002', () => {
...
+                vendorGstinInfo.trade_name,
+                clientGstinInfo.trade_name
+                    ? clientGstinInfo.trade_name
+                    : clientGstinInfo.value
             );
+            await vendorOnboarding.gstinInfoCheck();
+            bankAccountNumber = await getBankDetails.bankAccountNumber();
         });
+
+        await test.step('Verify Client Connect Details', async () => {});
```
Comment (sirjandrn25) [inline src/tests/Vendor_Onboarding/initial_gstvendormanaged_vowithclientconnect.spec.ts:114]: why this is empty

File: playwright.config.ts (hunk 1 above)
Comment (bibas07) [reply playwright.config.ts:6]: okay

File: src/tests/Vendor_Onboarding/initial_gstvendormanaged_vowithclientconnect.spec.ts (hunk 2 above)
Comment (bibas07) [reply src/tests/Vendor_Onboarding/initial_gstvendormanaged_vowithclientconnect.spec.ts:114]: I removed code that was used for checking visibility of GSTIN 
line 110: await vendorOnboarding.gstinInfoCheck(); do this task

Comment (sirjandrn25) [reply src/tests/Vendor_Onboarding/initial_gstvendormanaged_vowithclientconnect.spec.ts:114]: ohk

**Pull request merged**